modules/azure/src/main/java/org/testcontainers/azure/AzureServiceBusEmulatorContainer.java

```diff
@@ -0,0 +1,107 @@
+package org.testcontainers.azure;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.MSSQLServerContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.LicenseAcceptance;
+
+/**
+ * Testcontainers implementation for Azure Service Bus Emulator.
+ * <p>
+ * Supported image: {@code mcr.microsoft.com/azure-messaging/servicebus-emulator}
+ * <p>
+ * Exposed port: 5672
+ */
+public class AzureServiceBusEmulatorContainer extends GenericContainer<AzureServiceBusEmulatorContainer> {
```

Comment:
Let's drop Emulator from class name.

```suggestion
public class AzureServiceBusContainer extends GenericContainer<AzureServiceBusEmulatorContainer> {
```